docs/releasing.md

```diff
@@ -0,0 +1,62 @@
+# Releasing
+
+We're releasing based on tags, which contain our version number. The format is 'v0.0.0'.
+The release title will be set to this version.
+
+The CI pipeline has a 'release' job, which will update the release on Github.
+That job triggers itself, when a draft release is created.
+
+## Create new release pipeline
+
+We release from release-branches. Each maintained release has a separate pipeline in Concourse.
+To create a new pipeline run this in the CI repository:
+
+```
+cd pipelines/cf-operator-release
+./configure.sh CFO v0.4.x v0.4
+```
+
+Where `CFO` is your concourse target and `v0.4.x` is the name of the branch.
+The last argument, `v0.4` is used to filter Github tags, which belong to the release.
+
+This allows a separate Github branch and Concourse pipeline for each major version.
+Within those pipelines, releases can be built from minor versions.
+
+## Create a new release
+
+After completion, the pipeline will create several artifacts:
+
+* helm chart on S3
+* cf-operator binary on S3
+* docker image of the operator on dockerhub
+
+Running the 'release' job will take the latest artificats, which passed through the pipeline and add them to the Github release:
+
+* to the body
+* as Github assets for downloading
+
+The version numbers (v0.0.0-build.SHA) of these assets are taken from the info on S3.
```

Comment:
To be precise the `build` part is actually the number of commits on top of the tag. So maybe change it to 

```suggestion
The version numbers (v0.0.0-<number-of-commits>.<commit-SHA>) of these assets are taken from the info on S3.
```
to be precise?

Reply:
Good point, I copied that blindly from the old docs.